src/app/globals.css

```
@@ -0,0 +1,505 @@
+  /* Breakpoints */
+  --breakpoint-sm: 640px;
+  --breakpoint-md: 768px;
+  --breakpoint-lg: 1024px;
+  --breakpoint-xl: 1280px;
+}
+
+:root {
+  font-size: 62.5%; /* 10px base */
+}
+
+@supports (font: -apple-system-body) and (-webkit-appearance: none) { 
```

Biome (2.1.2)

[error] Unexpected shorthand property font after font-size



(lint/suspicious/noShorthandPropertyOverrides)